libsofia-sip-ua/nta/nta.c

```diff
@@ -4316,6 +4319,7 @@ nta_leg_t *nta_leg_tcreate(nta_agent_t *agent,
   su_home_t *home;
   url_t *url;
   char const *what = NULL;
+  sip_via_t *via = NULL;
```

Comment:
```suggestion
  const sip_via_t *via = NULL;
```

To fix the following build error with gcc 14:
```
nta.c: In function 'nta_leg_tcreate':
./../sip/sofia-sip/sip_tag.h:340:57: error: passing argument 1 of 'siptag_via_vr' from incompatible pointer type [-Wincompatible-pointer-types]
  340 | #define SIPTAG_VIA_REF(x) siptag_via_ref, siptag_via_vr(&(x))
      |                                                         ^~~~
      |                                                         |
      |                                                         sip_via_t ** {aka struct sip_via_s **}
```

Reply:
That is done, thanks!